<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ventana</class>
 <widget class="QWidget" name="Ventana">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pbDireccion">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDireccion">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Direccion</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pteMostrar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>401</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leBusqueda">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Ingrese una busqueda</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTiempo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>141</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Tiempo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDistancia">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>171</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Distancia</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePartida">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>81</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Origen</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leLlegada">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>111</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Destino</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>81</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Desde :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>111</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Hasta :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>141</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Demora :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Distancia :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
